# Since we already have a bytecode patch, this one disables autohinting so that fonts in ooo won't look like epic fail
# By ilikenwf/Matt Parnell <parwok@gmail.com>

--- include/freetype/freetype.h	2009-07-13 11:51:20.871463424 -0500
+++ include/freetype/freetype.h.mod	2009-07-13 11:53:14.812970517 -0500
@@ -2456,7 +2456,7 @@
 #define FT_LOAD_RENDER                       0x4
 #define FT_LOAD_NO_BITMAP                    0x8
 #define FT_LOAD_VERTICAL_LAYOUT              0x10
-#define FT_LOAD_FORCE_AUTOHINT               0x20
+#define FT_LOAD_NO_AUTOHINT                  0x20
 #define FT_LOAD_CROP_BITMAP                  0x40
 #define FT_LOAD_PEDANTIC                     0x80
 #define FT_LOAD_IGNORE_GLOBAL_ADVANCE_WIDTH  0x200
@@ -2464,7 +2464,7 @@
 #define FT_LOAD_IGNORE_TRANSFORM             0x800
 #define FT_LOAD_MONOCHROME                   0x1000
 #define FT_LOAD_LINEAR_DESIGN                0x2000
-#define FT_LOAD_NO_AUTOHINT                  0x8000U
+#define FT_LOAD_FORCE_AUTOHINT               0x8000U
 
   /* */
 
